src/button/_button.scss

```diff
@@ -14,292 +14,186 @@
  * limitations under the License.
  */
 
-@import "../variables";
-@import "../mixins";
-
-// The button component. Defaults to a flat button.
 .mdl-button {
-  background: transparent;
+  min-height: 36px;
   border: none;
-  border-radius: $button-border-radius;
-  color: $button-secondary-color;
+  min-width: 88px;
   position: relative;
-  height: $button-height;
-  margin: 0;
-  min-width: $button-min-width;
-  padding: 0 $button-padding;
-  display: inline-block;
-  @include typo-button();
+  color: rgb(0, 0, 0);
+  background: transparent;
+  border-radius: 2px;
+  /*
+  Add 6 top and bottom padding.
+  Provides 12 total for a 48px high touch area.
+  */
+  padding: 6px 16px;
+  font-family: $performance-font;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: inline-flex;
+  font-size: 0.875rem;
```

Comment:
Should we use a SASS function like we'd discussed?
```
font-size: spToRem(14);
```

I'd prefer to keep this readable in terms of `sp`, which is what the spec defines. This would allow for easier audits and easier bugfixing in general.

Another option would be to add a comment explaining how we arrived at the value, e.g.:
```
font-size: 0.875rem; /* 14 sp */
```